src/engine/Operation.cpp

```diff
@@ -574,6 +570,21 @@ const VariableToColumnMap& Operation::getInternallyVisibleVariableColumns()
   return variableToColumnMap_.value();
 }
 
+// _____________________________________________________________________________
+absl::Cleanup<absl::cleanup_internal::Tag, std::function<void()>>
+Operation::resetChildLimitsAndOffsetOnDestruction() {
+  std::vector<std::pair<QueryExecutionTree*, LimitOffsetClause>>
+      childrenClauses;
```

Comment:
For good practice, this can be `absl::InlinedVector<3>` to avoid allocations (for good practice).